Automattic/jetpack
# VideoPress: start to migrate block from Jetpack to VideoPress package

This PR starts the migration process of the VideoPress (beta) block of the Jetpack plugin to the VideoPress package.
* VideoPress package contains the code that defines the VideoPress block
* VideoPress package API exposes the `register_videopress_block()` method to delegate the block registering process.
* VideoPress plugin registers the VideoPress block
* Jetpack does not register the new VideoPress block. Follow-up PR is coming up for this
* VideoPress v6 is still working on the Jetpack plugin. Cleaning in a follow-up PR.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: start to migrate block from Jetpack to VideoPress package

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Compile / update / etc Jetpack products dependencies. Probably, we'd like to run...

```
jetpack build --all
```

* In your WordPress testing site, go to the plugins admin page
* Activate VideoPress plugin
* Go to Block Editor
* Create/edit a site post
* Create a new VideoPress block
* Pay attention to the blocks there:
  * VideoPress block is the block registered by the package
  * VideoPress (beta) is the block registered by the Jetpack plugin
<img width="370" alt="image" src="https://user-images.githubusercontent.com/77539/183080129-aeb5f8c9-d8f5-4aa8-b706-783f3c9abb94.png">

Upload a video using the `VideoPress` block. Confirm it works as expected.




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202561463041784
